### DISCUSSION TITLE
How to render only one remote participant without displaying local participant?

### DISCUSSION THREAD

**Discussion by eddielin0926** [Azure/communication-ui-library#2632] [Q&A] [answered]
I am trying to use ACS like watching video streaming. Therefore, I don't need the local participant `VideoTile`. 
However, I cannot find a proper way to hide local participant while using `VideoGallery` or `CallComposite` .
Should I use `VideoTile` with the method render element along with `StreamMedia`, or is there any other recommended approach?

**Comment by JamesBurnside** [comment, marked as answer]
Hi @eddielin0926, you are correct; today you cannot hide the local preview from the `CallComposite` or `VideoGallery`. The reason this is so is for privacy concerns, we _never_ want a user's camera to be active in a call but not have the user be aware of it. To the same privacy concern, the local tile also informs the user of their microphone's muted/unmuted status. This is great feedback feedback however and your scenario is very valid - we are looking into how we could still enforce a good level of privacy assurances while providing support for hiding the local preview.

You can build out your own VideoGallery experience using, as you mention, `VideoTile` with `StreamMedia` components. A good starting place for building this is to take a look at the code in [RemoteVideoTile](https://github.com/Azure/communication-ui-library/blob/release/1.4.1/packages/react-components/src/components/RemoteVideoTile.tsx).
